front/cypress/e2e/profile_edition.cy.ts

```diff
@@ -0,0 +1,43 @@
+import { randomString, randomEmail } from '../support/commands';
+
+describe('profile edition', () => {
```

Comment:
This tests the very basic functionality. Is there more cases we should cover? Maybe the password change?